### PR TITLE
Update email instructions to counsel patience, contact support

### PR DESCRIPTION
**Pull request by cbf66**

### CODE REVIEW

File: source/help/registerhelp.md [hunk 1]
```diff
@@ -35,4 +35,7 @@ If you need to change your preferred e-mail address, for example because
 you have changed institutions, then you should visit the [e-mail change
 form](https://arxiv.org/auth/email-change-form). This will ask for your new e-mail
 address and will send an e-mail with a verification code to your new
-address to make sure that your new address is correct.
+address to make sure that your new address is correct. Please allow up to an hour
+for delivery of the verification code, before requesting a new one. If you try two
+times and still haven't recieved a verification code, or your verification code
```
Comment (jimentwood) [inline source/help/registerhelp.md:40]:
```suggestion
times and still have not received a verification code, or your verification code 
```

File: source/help/registerhelp.md [hunk 2]
```diff
@@ -35,4 +35,7 @@ If you need to change your preferred e-mail address, for example because
 you have changed institutions, then you should visit the [e-mail change
 form](https://arxiv.org/auth/email-change-form). This will ask for your new e-mail
 address and will send an e-mail with a verification code to your new
-address to make sure that your new address is correct.
+address to make sure that your new address is correct. Please allow up to an hour
+for delivery of the verification code, before requesting a new one. If you try two
+times and still haven't recieved a verification code, or your verification code 
+doesn't work, please contact [arXiv user support](https://arxiv.org/support).
```
Comment (jimentwood) [inline source/help/registerhelp.md:41]:
```suggestion
does not work, please contact [arXiv user support](https://arxiv.org/support).
```

**Review by jimentwood — CHANGES_REQUESTED** [review]
looks good with a couple minor changes

**Review by jweiskoff — APPROVED** [review]
Looks good with the changes.

**Pull request merged**